### PR TITLE
added deployment config for this-or-that

### PR DESCRIPTION
**Pull request by jordilaforge**

### CODE REVIEW

File: apps/okd4-this-or-that-apps.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#  - groupName: "team1"
+#    clusterRoleName: "admin"
+
+clusterResourceWhitelist:
+
+namespaceResourceBlacklist:
+#  - group: 'apps'
+#    kind: 'DaemonSet'
+
+applications:
```
Comment (niiku) [inline apps/okd4-this-or-that-apps.yaml:23]: `applications` should be managed through the tekton job - not entered manually

**Review by niiku — COMMENTED** [review]
You also need to modify the `bootstrap/values.yaml` file

**Comment by niiku** [comment]
Don't forget the following push webhooks:
```
https://argocd.baloise.dev/api/webhook
https://tekton-syncapps-route-tekton-chatopshandler.apps.okd.baloise.dev/
```

File: apps/okd4-this-or-that-apps.yaml (hunk 1 above)
Comment (jordilaforge) [reply apps/okd4-this-or-that-apps.yaml]: done

File: apps/okd4-this-or-that-apps.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#argocdNamespace: "argocd"
+#server: "https://kubernetes.default.svc"
+namespace:
+  annotations: {}
+  labels:
+    client: jordilaforge
+
+enableDefaultNetworkPolicy: false
+enableDefaultLimitRange: false
+allowAllNamespaces: true
```
Comment (niiku) [inline apps/okd4-this-or-that-apps.yaml]: pls set to false

**Review by niiku — APPROVED** [review]

**Pull request merged**